tests/unit/resource/CMakeLists.txt

```diff
@@ -15,6 +17,8 @@ set(tests
     suspend_thread_timed
 )
 
+set(cross_pool_injectionr_PARAMETERS THREADS_PER_LOCALITY all)
+set(scheduler_priority_check_PARAMETERS THREADS_PER_LOCALITY all)
```

Comment:
Should we keep it at `4` instead of `all`?

Reply:
the cross pool test needs as many threads as possible to be useful, so I'd prefer all, the scheduler check doesn't matter much but all gives a more interesting output